src/wasm-s-parser.h

```diff
@@ -279,12 +279,17 @@ class SExpressionWasmBuilder {
   Expression* makeRttCanon(Element& s);
   Expression* makeRttSub(Element& s);
   Expression* makeRttFreshSub(Element& s);
+
+  // A field may be packed, in which case it is signed or unsigned, or it may
+  // not be packed.
+  enum class Packed { Signed, Unsigned, Non };
```

Comment:
Maybe `Extension` would be more descriptive than `Packed` here? If so, I would also switch `Non` to `None`.

Reply:
Hmm, I actually had to read your sentence twice as my brain thought you meant something to do with a suffix (like a file extension). But fair point that extending is the actual operation here. Anyhow, I think I prefer it as it is, but I don't feel strongly.

Reply:
Perhaps "PackedExtend"? Seems verbose though.

Reply:
Maybe `Extend` would be better than `Extension`? It's too bad there's not a better word describing both sext and zext.

Reply:
Or maybe `PackedRead` or `PackedReadKind`? It's not really the packed field that is signed, but the read operation itself.

Reply:
Hmm, I'm not that happy with any of these names I guess. Maybe the name could refer to "signedness" more generally, as in "this is signed / this is unsigned / signedness does not matter here" - ?

Reply:
`Signedness` sgtm and could probably replace some bools elsewhere in our code base.

Reply:
Good point, yes - I think that is the right direction. It feels correct to replace the various `bool signed_` with something like that. PR updated, and if that makes sense further refactoring could be done later.